src/page/Landing/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Button, Typo } from '@scrpoker/components';
+import Style from './style.module.scss';
```

Comment:
Mostly Pascal case is for Class name, but this is object, so:
```suggestion
import style from './style.module.scss';
```